EventListener/AttachRefreshTokenOnSuccessListener.php

```diff
@@ -72,7 +84,9 @@ public function __construct(
         $singleUse,
         RefreshTokenGeneratorInterface $refreshTokenGenerator,
         ExtractorInterface $extractor,
-        array $cookieSettings
+        array $cookieSettings,
+        $returnExpiration,
+        $returnExpirationParameterName
```

Comment:
Adding new required constructor params is a B/C break, they must be optional.

```suggestion
        bool $returnExpiration = false,
        string $returnExpirationParameterName = 'refresh_token_expiration'
```

Reply:
√